app/models/shipit/deploy_spec/npm_discovery.rb

```diff
@@ -3,6 +3,15 @@
 module Shipit
   class DeploySpec
     module NpmDiscovery
+      # https://docs.npmjs.com/cli/publish
+      PUBLIC = 'public'
+      PRIVATE = 'restricted'
+      VALID_ACCESS = [PUBLIC, PRIVATE]
+
+      NPM_REGISTRY = "https://registry.npmjs.org/"
```

Comment:
Using the public npm registry for public publishes since yarnkpkg is just a reverse proxy to the npm registry.